Documentation/Evolution/StronglyTypedCaptures.md

```diff
@@ -221,7 +222,7 @@ matched by its contained pattern. A capturing group's capture type is
 ```swift
 let graphemeBreakLowerBound = /([0-9a-fA-F]+)/ // => `Regex<Substring>`
 
-// Result builder DSL equivalent
+// Equivalent result builder syntax:
```

Comment:
I know that this should be pretty clear to the reader, but this one is not really using the result builder syntax...  That's why I was using "DSL" to refer to anything that involves types like `OneOrMore`.